tests/testthat/test_inspectMetaDifferences.R

```diff
@@ -5,13 +5,18 @@ load(file = "helper_data.rda")
 test_that("Errors",{
   df1_5 <- df1_4 <- df1_2 <- df1_3 <- df1
   expect_error(inspectMetaDifferences(c("1", "2"), df1, df1_2),
-               "'varName' must be a character of length 1.")
+               "'varName' needs to be a character vector of length 1.")
 })
 
 test_that("Compare two identical GADSdat objects",{
   out <- inspectMetaDifferences("V1", df1, df1)
   expect_null(out$varDiff)
   expect_null(out$valDiff)
+
+  out2 <- inspectMetaDifferences("V1", "helper_dataBase.db",
+                                "helper_dataBase.db")
```

Comment:
This is due to the working directory during tests being `.../eatGADS/tests/testhat` but interactively it is `.../eatGADS` (because of the RStudio project). Therefore, I often have two lines in each file for loading data, with the interactive on being commented out. Maybe this could be done more elegantly, but this is my work around :P

Reply:
Ah, I see! I did not have such a case, yet. But I hope I remember this workaround when it comes to such a test case 👍